src/sentry/api/endpoints/project_details.py

```diff
@@ -240,13 +241,24 @@ def validate_symbolSources(self, sources_json):
         except InvalidSourcesError as e:
             raise serializers.ValidationError(str(e))
 
-        sources_json = json.dumps(sources) if sources else ""
-
         # If no sources are added or modified, we're either only deleting sources or doing nothing.
         # This is always allowed.
         added_or_modified_sources = [s for s in sources if s not in orig_sources]
         if not added_or_modified_sources:
-            return sources_json
+            return json.dumps(sources) if sources else ""
+
+        # All modified sources should get a new UUID, as a way to bust caches.
+        # For example, symbolicator caches negative DIF lookups based on this id,
+        # and updating a source should take effect immediately instead of an
+        # hour later when caches expire.
```

Comment:
IIUC symbolicator takes this ID directly and puts it in the cache key: https://github.com/getsentry/symbolicator/blob/dd01a362c627afe7d5fbac5098c548ac637b81dc/crates/symbolicator/src/services/download/locations.rs#L142-L161

I'd like to see that a little more explicitly called out maybe, because I had to check and look that up.